Scripts/gen_Mix.sh

```diff
@@ -261,6 +261,8 @@ echo services.DbService.purpose: ${CAMPAIGN}_${DBPURPOSE} >> mix.fcl
 echo services.DbService.version: ${DBVERSION} >> mix.fcl
 echo services.DbService.verbose : 2 >> mix.fcl
 echo "services.GeometryService.bFieldFile : \"${FIELD}\"" >> mix.fcl
+echo "#include \"Production/JobConfig/digitize/MakeSurfaceSteps.fcl\"">> mix.fcl
```

Comment:
If we're adding SurfaceSteps for mixing they shoudl also be in digitize.

Reply:
I think this is already in the gen digi script if thats what you mean?